src/components/ui/UserInfo.tsx

```diff
@@ -58,13 +69,26 @@ export const UserInfo = ({ user }: Props): JSX.Element => {
     await unfollowUser({ targetUserId: user.userId });
   };
 
+  const handleBan = async (): Promise<void> => {
+    await banUser({ targetUserId: user.userId });
+    refetch()
```

Comment:
Like a temporary decision is ok, but is better to config cache for the endpoint, because ban method return list of banned users by himself